doc/source/whatsnew/v0.20.0.txt

```diff
@@ -589,3 +589,5 @@ Bug Fixes
 - Bug in ``Series.replace`` and ``DataFrame.replace`` which failed on empty replacement dicts (:issue:`15289`)
 - Bug in ``pd.melt()`` where passing a tuple value for ``value_vars`` caused a ``TypeError`` (:issue:`15348`)
 - Bug in ``.eval()`` which caused multiline evals to fail with local variables not on the first line (:issue:`15342`)
+
```

Comment:
move into a blank space above so that you don't get conflicts.

``with converting a datetime64 index``